### ISSUE TITLE
Check labelling for logging in and signing up users

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4080]
There are two ways labels are handled in the platform, depending on the markup of the form:

```
<label for="username">Username</label>
<input id="username" type="text" />
```
and

```
<label>Username
     <input id="username" type="text" />
</label>
```

Go through the following forms and make sure the element is either using the for-attribute to associate the form-control or is encapsulating the form-control like the second example.

The login and sign-up forms are found when clicking on Login and Signup in the lefthand mode-bar:
<img width="75" alt="Screenshot 2020-10-05 at 12 20 20" src="https://user-images.githubusercontent.com/8624777/95068239-522f6200-0705-11eb-8cad-d03e1febfaf5.png">

The forgot-password-form is found when clicking on "Forgot password" in the login-form:
<img width="344" alt="Screenshot 2020-10-05 at 12 22 15" src="https://user-images.githubusercontent.com/8624777/95068441-96226700-0705-11eb-86c1-ca643dff405b.png">

The files related to the above form is found here:
- https://github.com/ushahidi/platform-client/blob/develop/app/common/auth/login.html
- https://github.com/ushahidi/platform-client/blob/develop/app/common/auth/password-reset-confirm.html
- https://github.com/ushahidi/platform-client/blob/develop/app/common/auth/password-reset.html
- https://github.com/ushahidi/platform-client/blob/develop/app/common/auth/register.html

If no adjustments is needed, just comment in this issue that you have worked through the issues and checked, if adjustment is needed, make the necessary changes and create a PR 🎉   If changes are needed, make sure the form still looks the same as before and that is has no effect on the visual appearance of the form.

Aha! Link: https://ushahiditeam.aha.io/features/PROD-836

**Comment by Akshansh99** [comment]
Can I work on this?

**Comment by Akshansh99** [comment]
I've checked all the mentioned files and no further adjustment is required.

**Comment by niksseif** [comment]
Hi there, 
I went through all the files mentioned above and all the labels have for attribute same as the id of input field.

**Comment by Angamanga** [comment]
@Akshansh99 @niksseif Thank you for your contribution 🎉 !

**Comment by Angamanga** [comment]
@all-contributors please add @Akshansh99 for ally

**Comment by Angamanga** [comment]
@all-contributors please add @Akshansh99 for a11y

**Comment by Angamanga** [comment]
@all-contributors please add @niksseif for a11y

**Comment by error404-sp** [comment]
Hi @Angamanga , 
I came across this project from the outreachy website and found it interesting. I went through the given links and found that some minimal adjustments can be made to improve accessibility.
I will like to be assigned this issue.
Could you please provide the docs to set up environment if it's necessary in this project.

**Comment by Angamanga** [comment]
@error404-sp Please go ahead :) Here is the docs for the environment: https://app.gitbook.com/@ushahidi/s/platform-developer-documentation/, check especially these pages for setting up the dev-environment:
https://app.gitbook.com/@ushahidi/s/platform-developer-documentation/development-and-code/setup_alternatives#platform-api-backend-install-guides

and

https://app.gitbook.com/@ushahidi/s/platform-developer-documentation/development-and-code/setup_alternatives#platform-client-frontend-install-guides

Let me know if you run into problems and need guidance!

**Comment by error404-sp** [comment]
Thank you @Angamanga. I will let you know. Unfortunately, the link 
https://app.gitbook.com/@ushahidi/s/platform-developer-documentation/development-and-code/setup_alternatives#platform-api-backend-install-guides
and 
https://app.gitbook.com/@ushahidi/s/platform-developer-documentation/development-and-code/setup_alternatives#platform-client-frontend-install-guides
 give "Page not found" errors. I went through the first link.
Just wanted to confirm if 
(backend)
https://docs.ushahidi.com/platform-developer-documentation/development-and-code/setup_alternatives#platform-api-backend-install-guides
and 
(frontend)
https://docs.ushahidi.com/platform-developer-documentation/development-and-code/setup_alternatives/setting-up-the-platform-client 
are the links you wanted to provide me(found it in the documentation link) or am I missing on any link for the setup.
Thanks

**Comment by error404-sp** [comment]
Hi @Angamanga ,
I have created the changes in the above files. Kindly review it.
https://github.com/ushahidi/platform-client/pull/1554#issue-499389488
I would really appreciate feedback on it.
I have refered to the W3C standards for the changes.
Thanks

**Comment by Angamanga** [comment]
@error404-sp Thank you for your contribution! I will review shortly!

**Comment by TunrayoIlawole** [comment]
Can I work on this?

**Comment by rowasc** [comment]
@TunrayoIlawole Hey. There's already someone working on it, so in this case, no. However you can check the other tickets in the hacktoberfest and outreachy labels. Thank you!

**Comment by tuxpiper** [comment]
@all-contributors please add @error404-sp for code
